src/components/backend-ai-session-list.ts

```diff
@@ -4412,9 +4413,35 @@ ${rowData.item[this.sessionNameField]}</pre
     this.refreshList();
   }
 
-  _updateImagifyAvailabilityStatus(e) {
-    this.canStartImagifying = e.target.validity.valid;
-    this.newImageName = e.target.value;
+  _validateImageName() {
+    this.newImageNameInput.validityTransform = (value, nativeValidity) => {
+      if (!nativeValidity.valid) {
+        if (nativeValidity.patternMismatch) {
+          this.newImageNameInput.validationMessage = _text(
+            'session.Validation.EnterValidSessionName',
+          );
+          return {
+            valid: nativeValidity.valid,
+            patternMismatch: !nativeValidity.valid,
+          };
+        } else {
+          this.newImageNameInput.validationMessage = _text(
+            'session.Validation.EnterValidSessionName',
+          );
+          return {
+            valid: nativeValidity.valid,
+            customError: !nativeValidity.valid,
+          };
+        }
+      } else {
+        this.canStartImagifying = true;
```

Comment:
`canStartImagifying` property is only used to check whether the push session button is enabled or not. How about removing it and checking `disabled` property of the push session button with `checkValidity()`?